addon/index.js

```diff
@@ -33,13 +33,7 @@ export default Mixin.create({
   updateScrollPosition (transitions) {
     const lastTransition = transitions[transitions.length - 1]
 
-    let routerPath
-    if (typeof get(lastTransition, 'handler._router') !== 'undefined') {
-      routerPath = 'handler._router'
-    } else {
-      routerPath = 'handler.router'
-    }
-    const url = get(lastTransition, `${routerPath}.currentURL`)
```

Comment:
This is a good refactor 👍

Reply:
@snewcomer Just wondering what the last version of Ember we support?

Reply:
I believe 2.12.